### PR TITLE
Support ESLint issues import

### LINKED ISSUE #1001: Support ESLint external issues report
See https://jira.sonarsource.com/browse/MMF-1231

### PR DESCRIPTION
**Pull request by vilchik-elena**
fixes #1001

### CODE REVIEW

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/eslint/react.json [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+  {
+    "key": "react/jsx-closing-tag-location",
+    "name": "Validate closing tag location in JSX (fixable)",
+    "url":
+    "https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-closing-tag-location.md"
+  },
+  {
+    "key": "react/jsx-curly-spacing",
+    "name":
+    "Enforce or disallow spaces inside of curly braces in JSX attributes and expressions (fixable)",
```
Comment (saberduck) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/eslint/react.json]: Should we have  "fixable" in the title?

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JsonProfileReader.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ private JsonProfileReader() {
   }
 
   public static Set<String> ruleKeys(String pathToFile) {
-    URL profileUrl = JsonProfileReader.class.getResource(pathToFile);
+    URL profileUrl = JsonProfileReader.class.getClassLoader().getResource(pathToFile);
```
Comment (saberduck) [inline sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JsonProfileReader.java]: I don't understand why this change was necessary?

File: its/plugin/projects/eslint_report/readme [hunk 3]
```diff
@@ -0,0 +1,4 @@
+The report was generated with
+npx eslint --ext ts -f json . > report.json
```
Comment (saberduck) [inline its/plugin/projects/eslint_report/readme]: nice to have: I didn't manage to get the same report as in the example, it would be good to have instructions how to get exactly same behavior. We can even generate two separate reports to test import of multiple reports

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JsonProfileReader.java (hunk 2 above)
Comment (vilchik-elena) [reply sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JsonProfileReader.java]: looks like `Class#getResource` and `ClassLoader#getResource` don't do the same thing and in analyzer-commons we use `ClassLoader#getResource`
